### ISSUE TITLE
vline sometimes breaks out of subplot

### ISSUE THREAD

**Issue by ssfrr** [JuliaPlots/Plots.jl#2214]
Drawing a `vline` in a subplot seems to sometimes break out and span the whole plot, instead of staying constrained to its subplot.

MWE:

```julia
plts = []
for _ in 1:8
    push!(plts, plot(randn(16)))
    vline!([rand()*15+1])
end

plot(plts...; layout=(4,2))
```

gives:
![image](https://user-images.githubusercontent.com/734386/66622743-e3cdc780-ebb6-11e9-8f20-391355784c0c.png)

It doesn't seem to do this if I use a smaller number of subplots.

This is with Plots v0.26.3.

**Comment by ssfrr** [comment]
It seems improved but still present. This is what I see now. Notice the line on the bottom right is still escaping its subplot:

![image](https://user-images.githubusercontent.com/734386/66657264-89138a80-ec0d-11e9-878a-68a77c3adb36.png)

**Comment by daschw** [comment]
Weird, I get:
![vline](https://user-images.githubusercontent.com/16589944/66658117-19b78e00-ec41-11e9-90f4-86a9b5010649.png)

Does this only happen in GR for you?

**Comment by ssfrr** [comment]
GR is the only backend I have installed. Also it only happens with SVG output, not PNG

**Comment by daschw** [comment]
Yep, I can reproduce with `fmt = :svg`.

**Comment by daschw** [comment]
@jheinen on Slack:
> This is a clipping problem in the GR SVG driver. I'll check this ...

**Comment by jheinen** [comment]
The problem is fixed in GR 0.41.5

**Comment by daschw** [comment]
I can confirm that it's fixed with GR 0.41.5.
I had to checkout GR#master though, the latest tagged release is 0.40.0

**Comment by ssfrr** [comment]
Great, thanks for looking into it @daschw and @jheinen!
